ucca/validation.py

```diff
@@ -110,7 +112,7 @@ def validate_non_terminal(self, linkage=False, multigraph=False):
             if len(s) > 1:
                 yield "Unit (%s) with multiple %s children (%s)" % (self.node_id, tag, join(e.child for e in s))
         if ETags.Function in self.incoming:
-            s = self.outgoing_tags.difference((ETags.Terminal, ETags.Punctuation))
+            s = self.outgoing_tags.difference((set.union({ETags.Terminal, ETags.Punctuation}, NON_SCENE, SUPP_FUNC)))
```

Comment:
Seems like there is a redundant pair of parentheses here. Remember, a `tuple` is parentheses with a comma but if there is no comma then `(x)` is equivalent to `x`. See also another case below.